velox/exec/Task.cpp

```diff
@@ -553,6 +553,20 @@ void Task::start(
   facebook::velox::process::ThreadDebugInfo threadDebugInfo{
       self->queryCtx()->queryId(), self->taskId_, nullptr};
   facebook::velox::process::ScopedThreadDebugInfo scopedInfo(threadDebugInfo);
+
+#define CHECK_EARLY_TASK_ERROR()                                            \
```

Comment:
Macros are hard to read. What's the motivation for using a macro here instead of a regular function / method?

Reply:
You are right. It seems that a function is more appropriate here. Will change.